clients/include/rocblas_arguments.hpp

```diff
@@ -93,14 +99,14 @@ struct Arguments
         ifs.read(trailer, sizeof(trailer));
 
         if(strcmp(header, "rocBLAS") || strcmp(trailer, "ROCblas"))
-            error();
+            error(" ");
```

Comment:
`error("beginning or ending signature");` can be used to describe that mismatch.

Reply:
Correct. Adding error message.